Prace_domowe/Praca_domowa7/Jamrozy_Bartosz/pd7.ipynb

```diff
@@ -0,0 +1,926 @@
+{
```

Comment:
<p>Może to wynikać ze zdefiniowanej liczby klastrów w GMM</p><p> </p><p>Finalne wyniki poprawne, jednak sprawdzając jest poczucie niepełnego zrozumienia tematu o czym świadczy szukanie optymalnej liczby klastrów, co może wynikać, z tego że zadanie było niesztampowe.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>